admin/bench-measure.mk

```diff
@@ -46,6 +46,15 @@ memory: $(BENCH)
 	$(MEMUSAGE) $^ memory TLS13_AES_256_GCM_SHA384 1000
 	$(MEMUSAGE) $^ memory TLS13_AES_256_GCM_SHA384 5000
 
+threads: $(BENCH)
+	for thr in 2 4 8 12 16 24 32 40 48 56 64 72 80 92 112 128; do \
```

Comment:
That seems like shitload of thread counts, is that worth it in terms of runtime? It doesn't seem like running at, say, non-powers of 2 will surface new information over above what we will get running only at 2 to the power 1..7.

Reply:
I think maybe this list is overfitting on 80 cores (certainly it is not very useful for me when run locally on a 16 core laptop) -- replaced with a python script which explains the goal